### DISCUSSION TITLE
Lightning symbol always on after uninstalling free version

### DISCUSSION THREAD

**Discussion by ghost** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#404] [Q&A]
Hi,
I uninstalled the free version with the intent of getting Pro, but haven't gotten to it yet. However, since uninstalling I've noticed that the lightening symbol is always on, even hours after the battery reaches 100%... now I'm a bit sketched out about downloading Pro!

-     MacBook Pro (13-inch, M1, 2020)
-     Big Sur 11.4
-     Aldente Free version
-     Did not use AlDente in Intel mode
-     Did not use the uninstaller, since I had the free version so I was told to uninstall as normal with no special steps needed
-     Already tried an SMC reset

Here's a screenshot of battery health management showing the lightning symbol on all night while plugged in. Any ideas for how to fix this and return the battery management back to normal?

<img width="427" alt="Screen Shot 2021-10-17 at 8 35 30 AM" src="https://user-images.githubusercontent.com/92031160/137614771-2e617f34-efd0-4ff4-aff4-b7f984ed39db.png">

Thanks in advance!

**Comment by MatthiasKerbl** [comment]
Hi @Lbobbing,

In my understanding, the lightning symbol just means that your MacBook is plugged in. It does not mean that it is charging necessarily.

Cheers,
Matthias

**Reply by ghost** [reply]
Hi Matthias,
Thanks for responding. Previously, when the battery was fully charged, the symbol would switch to a plug. Now the lightening bolt is always on, which I'm worried indicates that the computer is confused and still trying to charge. Here are the two symbols I'm referring to.
Thanks again!
![Screen Shot 2021-10-22 at 12 02 21 PM](https://user-images.githubusercontent.com/92031160/138435538-df3c8690-218e-44c8-a0e9-40e1f6d8c251.png)

**Reply by MatthiasKerbl** [reply]
Sorry for the misunderstanding, I thought you were talking about the lightning symbol on your screenshot.
Regarding the menubar icon, I have not seen this behavior yet. What does status if you do a "right-click" on the battery icon? Does it says that it still charges or "Battery is not charging"?

Cheers,
Matthias

**Reply by ghost** [reply]
No problem, I was also asking about the lightening symbol in the first screenshot (which I think also used to differentiate between "plug" and "lightening" icons).

Regarding the menubar icon, here's what I see ("fully charged" and "power source: adapter"). But no plug icon!

Just as a test, I was considering downloading AlDente again and setting the charge cap to 99% instead of 100% before uninstalling, but I haven't tried this yet. Let me know if you have any theories.

Thanks again!

<img width="184" alt="Screen Shot 2021-10-22 at 3 08 37 PM" src="https://user-images.githubusercontent.com/92031160/138458882-60fe75e5-aff6-4004-9c2b-a65e43652b41.png">

**Reply by MatthiasKerbl** [reply]
Thank you for clarifying. Your MacBook definitely knows that it is fully charged but for some reason, it doesn’t change the icon. One possibility could be that it was still balancing the battery but this should be done after a couple of hours and yours is already plugged in for longer than that.
I couldn’t really find anything on this issue and I don’t know if it is related to AlDente. You can definitely try to reinstall AlDente and maybe it will change something back.
If I find anything, I will let you know!

Cheers,
Matthias

**Reply by ghost** [reply]
Hi Matthias,
I haven't been able to find anything either, but timing-wise I am pretty sure it is somehow or another related to AlDente (I started this computer out of the box, didn't restore from a backup or anything). If I figure it out I will also let you know! Thanks again.
Cheers

**Reply by macubukujkun** [reply]
Did you find a solution?

**Reply by ghost** [reply]
After updating to Monterey the issue seems to have stopped. The builtin charge limiter will sometimes kick in and limit charging at 80%. It keeps trying to outsmart me, so I may still go for AlDente premium, but I think this is Apple's normal battery optimization algorithm and not related to AlDente.

Also, I notice that when it isn't charge-limiting, the lightening bolt stays on even when the battery is at 100% - I only see the plug symbol when the builtin charge limiter is working. So I think that aspect of my original issue may be normal behavior.

**Reply by macubukujkun** [reply]
I'm a foreigner. I want to make sure I understand correctly. If it's 100%, you're still the lighting icon, right?

**Reply by ghost** [reply]
Yes, if it's 100% the lightning icon is still on.

The only time this is NOT the case is when the Apple's charge limiter decides it's time to discharge the battery. Then the plug symbol comes on, and the numbers slowly go down to 80%, and stay there until Apple predicts I want it to go back up to 100% (which is never what I want).

**Reply by macubukujkun** [reply]
I am using the 2015 MacBook Pro. There is no function of waiting at 80% and charging to 100%.
